index.js

```diff
@@ -4,12 +4,13 @@ module.exports = {
   env: {
     es6: true,
     node: true,
-    browser: true
+    browser: true,
+    jest: true
   },
   parserOptions: {
     parser: "babel-eslint",
+    "ecmaVersion": 9,
```

Comment:
why 9 instead of 10? Here it indicates we can have up to version 10: https://eslint.org/docs/user-guide/configuring